concepts/60 Themes and Styles/05 Predefined Themes/50 Apply a Theme/05 Using SCSS.md

```diff
@@ -0,0 +1,47 @@
+You can import the theme bundle from the `bundles` folder from the `devextreme` package in the root `.scss` file:
+
+    /* @use 'devextreme/scss/bundles/dx.theme_name.scss';  */
+    @use 'devextreme/scss/bundles/dx.carmine.scss';
+
+Or you can import individual files for the whole theme. The example below imports files of the `Material Blue Light Compact` theme
+
+    @use 'devextreme/scss/widgets/material/colors' with ($color: 'blue', $mode: 'light');
+    @use 'devextreme/scss/widgets/material/sizes' with ($size: 'compact');
+    @use 'devextreme/scss/widgets/material/icons';
+    @use 'devextreme/scss/widgets/material/widget';
+    @use 'devextreme/scss/widgets/material/card';
+    @use 'devextreme/scss/widgets/material/fieldset';
+    @use 'devextreme/scss/widgets/material/common';
+    @use 'devextreme/scss/widgets/base/resizable';
+    @use 'devextreme/scss/widgets/base/draggable';
+    @use 'devextreme/scss/widgets/base/ui';
+    @use 'devextreme/scss/widgets/material';
+
+The theme path can be either `material` or `generic` depending on the theme family. The second solution is useful if you need to import only specific components. To do this, replace `@use 'devextreme/scss/widgets/material';` with `@use 'devextreme/scss/widgets/componentName';`:
```

Comment:
> The theme path **can be** either `material` or `generic` depending on the theme family.

But that's not quite true. `material` or `generic` is only a part of the path, not the entire path itself.

> The second solution is useful if you need to import only specific components.

What "second solution" you are referring here to? I do not understand. If you meant this

> Alternatively, you can import individual files that comprise a theme.

You mentioned it way too far away from this point; therefore, it's impossible to connect it with the current context.

> The second solution is useful if you need to import only specific components. **To do this**, replace `@use 'devextreme/scss/widgets/material';` with `@use 'devextreme/scss/widgets/componentName';`:

Do what? This entire paragraph feels a bit out-of-place here. We definitely need to do something about it.